### ISSUE TITLE
my transaction is not confirming

### ISSUE THREAD

**Issue by Aristarh787** [spesmilo/electrum#8774]
I'm trying to withdraw bitcoins, the transaction freezes and is constantly being processed, and so on endlessly. What is the reason? The program has been updated to the latest version. On the old version I can withdraw a little money, but on the new version they don’t withdraw at all.
How can I withdraw my money?

**Comment by SomberNight** [comment]
Hard to understand what you mean. Please include screenshots.

**Comment by Aristarh787** [comment]
The money is not transferred to another wallet, I can’t withdraw money at all, I’ve been trying for a month now. Just in the same form as in the photo and that’s it. How can I withdraw money? My transaction hangs and that’s it! I increased the commission and still can’t send it! I need my money, why can't I withdraw it?
![skrin](https://github.com/spesmilo/electrum/assets/154636868/cc7f83bb-e461-4320-813e-d337c11c6a80)

**Comment by SomberNight** [comment]
There are a lot of transactions in the bitcoin mempool atm and it looks like you are using low fees (compared to what others are willing to pay).
Try bumping the fee. Or if it is not urgent, you can just wait.
See https://bitcoinelectrum.com/frequently-asked-questions/#my-bitcoin-transaction-is-not-confirming-what-can-i-do

**Comment by Aristarh787** [comment]
I raised my commission, how much should I raise it? If I send 30 dollars, I should choose a commission of 30 dollars, since I already set a commission of 10 dollars when transferring 30 dollars, the payment also does not arrive, the transaction is frozen, the bitcoins have not been sent for more than a week, why should I wait another month? The most interesting thing is that it is not sent, and if canceled there, the transaction fee is immediately charged. Money is not sent and only from Electro’s wallet, I have wallets on different services and everything works everywhere, but Electro, like thieves, takes a commission for a transfer that never happened!

**Comment by AlexSel1** [comment]
Не работайте с Electrum. Они воры и обманьшики!

**Comment by Aristarh787** [comment]
> Не работайте с Electrum. Они воры и обманьшики!

Знал бы я это раньше, полтора года вроде поюзал кошелек и проблем не было, а сейчас не знаю что случилось, наверное реально обманщики и мошенники если я так понял что не один такой! Эх. Надежда умирает последней.

**Comment by SomberNight** [comment]
> I have wallets on different services and everything works everywhere

What other "services" do you use where you have wallets?
They are probably custodial services, like banks, managing the wallet for you. Potentially covering the fees out of pocket.
Electrum is a free open source non-custodial Bitcoin wallet. It is *not* a managed service. It is just a tool, like gimp, inkscape, libreoffice, or firefox. It is up to the user to do (and pay for) everything.
Unlike with a custodial/managed wallet, it is only the user who has access to their wallet. The developers or anyone else cannot see/move/access the coins. The fee that you pay when creating transactions also does not go to the developers. It goes to the bitcoin miners, who prioritise transactions based on who pays more.
It is convenient to claim the Electrum developers are thieves... but it just shows how you don't understand how it works at all.

What changed, is that in the last ~7 weeks the bitcoin mempool has been consistently full and some users (and bots...) have deep pockets and have been paying large fees to get their own bitcoin transactions mined. That is, it is now (once again) expensive to make onchain bitcoin transactions. (see e.g. https://jochen-hoenicke.de/queue/#BTC,6m,weight)
Basically if you only have, say, 100 usd worth of btc in your wallet, it is pretty much unusable atm, because it costs most of your money to make just one transaction. This is a problem with Bitcoin as a whole, not just Electrum.

You can use Lightning to make payments though, which Electrum also supports. However, merchants also need to support it, and, to start using Lightning initially, you need to open a channel, which too is an expensive onchain transaction.

**Comment by Aristarh787** [comment]
I withdrew the money, with a delay of a month, or maybe more, the commission had to be increased by an order of magnitude, of course.
